images/runtime/Dockerfile

```diff
@@ -8,7 +8,7 @@ ARG GOLANG_IMAGE=docker.io/library/golang:1.18.3@sha256:b203dc573d81da7b3176264b
 ARG UBUNTU_IMAGE=docker.io/library/ubuntu:20.04@sha256:626ffe58f6e7566e00254b638eb7e0f3b11d4da9675088f4781a50ae288f3322
 
 ARG CILIUM_LLVM_IMAGE=quay.io/cilium/cilium-llvm:547db7ec9a750b8f888a506709adb41f135b952e@sha256:4d6fa0aede3556c5fb5a9c71bc6b9585475ac9b1064f516d4c45c8fb691c9d9e
-ARG CILIUM_BPFTOOL_IMAGE=quay.io/cilium/cilium-bpftool:dd29c55c5ecad0ead395cb44c8a5c8ac67918909@sha256:2e566c175428ada5bac115b5ada42fedad6291250c5c963c6743922621659f34
+ARG CILIUM_BPFTOOL_IMAGE=quay.io/cilium/cilium-bpftool:06c0da890293b74a7af53dc9e2c48c4bf3350770@sha256:70d7ec0676498d9b62908ec04347ae5789c1b2a0b417803a9cb28755fd66de3b
```

Comment:
understand that this image is coming from master run of image-tools, just a note that this bpf-tool is now based on ubuntu 22.04 as per https://github.com/cilium/image-tools/pull/177, do you foresee any potential issue ? 

There is pending PR to bump base image to ubuntu 22.04 https://github.com/cilium/cilium/pull/19852

Reply:
In other words, we need to wait for https://github.com/cilium/cilium/pull/19852 to land, then rebase, right? Anything blocking the latter?

Reply:
We also need to update bpftool to fix the GKE CI jobs, so I'm starting to wonder if we should temporarily revert https://github.com/cilium/image-tools/pull/177. I'll take a look at the failing tests in https://github.com/cilium/cilium/pull/19852 today.

Reply:
I am completely agreed if we need to revert https://github.com/cilium/image-tools/pull/177, as ubuntu 22.04 might cause some more cycles, which could delay the release.